charts/datadog/values.yaml

```diff
@@ -520,6 +520,12 @@ datadog:
     # datadog.systemProbe.runtimeCompilationAssetDir -- Specify a directory for runtime compilation assets to live in
     runtimeCompilationAssetDir: /var/tmp/datadog-agent/system-probe
 
+    # datadog.systemProbe.enableCORE -- Enables Compile Once - Run Everywhere (CO-RE) for eBPF probes
+    enableCORE: true
```

Comment:
I see that it is enable by default. 
what happens if the host kernel version doesn't support CO-RE?
does the customer needs to change manually this Option to `false`?

do we have some logic in system-probe to fallback on the previous method?

Reply:
If CO-RE fails, we automatically fall back to runtime compilation (or whatever the current default way is of loading a particular eBPF program). You can see an example here: https://github.com/DataDog/datadog-agent/pull/14096

Reply:
FWIW - we enable CO-RE by default in the agent code as well.